src/js/components/DataTable/Resizer.js

```diff
@@ -131,9 +149,52 @@ const Resizer = ({ onResize, property, headerText, messages }) => {
           margin={{ top: 'xsmall' }}
           ref={ref}
           responsive={false}
-          onMouseDown={onMouseDown}
-          onMouseMove={start !== undefined ? onMouseMove : undefined}
-          onMouseUp={start !== undefined ? onMouseUp : undefined}
+          onMouseDown={onResizeStart}
+          onMouseMove={start !== undefined ? onResizeMove : undefined}
```

Comment:
do we actually need these since we have the event listeners above?

Reply:
I'm thinking that one of the 2 is redundant and I'm leaning towards remove the document event listeners and instead just leveraging these here. Will do some testing, just didn't want to remove anything since it was there before